src/payment/strategies/amazon-maxo/amazon-maxo-script-loader.ts

```diff
@@ -0,0 +1,31 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+import PaymentMethod from '../../payment-method';
+
+import { AmazonMaxoHostWindow, AmazonMaxoRegions, AmazonMaxoSDK } from './amazon-maxo';
+
+export default class AmazonMaxoScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        private _window: AmazonMaxoHostWindow = window
+    ) {}
+
+    load(method: PaymentMethod): Promise<AmazonMaxoSDK> {
+        const {
+            initializationData: { region = 'us' } = {},
```

Comment:
can't you do
```js
initializationData: { region = AmazonMaxoRegions.us } = {},
```

to avoid the casting to any?

It also seems that this value should be `na` by checking your enum

Reply:
Default Value 'us' is used to get 'na' form Enum if it didn't receive something from backend, depending on what it is receiving from backend is the value that is going to be on URL (de = 'eu', jp = 'fe', uk = 'eu', us = 'na').
We cannot skip from cast because initializationData is treated as an any, so in order that typescript compile we need to cast to any.